subset/pentests/test_nmap

```diff
@@ -66,7 +68,7 @@ rm -f .fail
 openportslist=" - "
 grep -oh '[0-9]*/open[^[:space:]]*' $LOG | while IFS=/ read -ra parts; do
     state=${parts[1]}
-    if [ "$state" == open ]; then
+    if [ "$state" == open ] || [ "$state" == "open|filtered" ]; then
```

Comment:
@grafnu please note this is intentional. Many BACnet devices show UDP port 47808 as filtered and it's useful to keep this condition so that port 47808 is shown in the report.

Reply:
This problem is that it can cause problems with other non-bacnet protocols. I had to change it as part of [this stability PR](https://github.com/faucetsdn/daq/pull/368/files#diff-b700bf6c0eae5d634645d492a283c594R69).  With this condition you added, what happens when there is an actual filtered port? It will be considered open, which means the test will fail.

This situation seems qualitatively different than the rest of the PR, so it might make sense to pull it out and address it independently because it sounds like it might be more than a one-line fix.